src/backend/storeManagers/sideload/games.ts

```diff
@@ -125,7 +126,7 @@ export async function uninstall({
 
   notify({ title, body: i18next.t('notify.uninstalled') })
 
-  removeShortcuts(appName)
+  removeShortcutsUtil(gameInfo)
```

Comment:
we already have the gameInfo here, no need to call the function that looks for the gameInfo